backend/tests_configs.ini

```diff
@@ -921,6 +921,15 @@ collaborative_document_edition.software = collabora
 collaborative_document_edition.collabora.backend.base_url = http://tracimbackendserver:8888
 collaborative_document_edition.collabora.base_url = http://localhost:9980
 
+[call_jitsi_test]
+app.enabled = contents/thread,contents/file,contents/html-document,contents/folder,upload_permission,share_content
+website.base_url = http://localhost:6543
+auth_types = internal
+call.enabled = True
+call.provider = jitsi_meet
+call.jitsi_meet.url = https://meet.jit.si
+call.unanswered_timeout = 20000
```

Comment:
why this config not use default value ?

Reply:
It's for test purpose, in this test we verify that we got the `call.unanswered_timeout` parameter with the value specified in test_configs.ini
In my opinion it is be better to verify another value than the default one since this is not a mandatory parameter. If the `call.unanswered_timeout` parameter is not present into the config file (development.ini) we just use the default value